src/realm/spec.cpp

```diff
@@ -257,6 +269,11 @@ bool Spec::convert_column_attributes()
                 break;
         }
```

Comment:
Should we give a dedicated error for switch selector values that are not listed above?